docker/Dockerfile.vlad

```diff
@@ -155,4 +155,4 @@ ENV WEBUI_PORT=7860
 
 EXPOSE ${WEBUI_PORT}
 ENTRYPOINT [ "/docker/entrypoint.sh" ]
-CMD python -u webui.py --listen --port ${WEBUI_PORT} ${CLI_ARGS}
+CMD python -u launch.py --listen --port ${WEBUI_PORT} ${CLI_ARGS}
```

Comment:
I purposefully don't use launch.py because it runs through and updates everything, clones repos, etc. which tends to blow out the image size & shouldn't be necessary for things to work. (it can also be overridden in the command line / dockerfile, if necessary)